packages/terra-form-validation/docs/NumericValidation.md

```diff
@@ -0,0 +1,169 @@
+
+The 2nd thing of note is that our validation function, `validateNumber`, will be given it's argument in the form of a string despite it being a number type input. This doesn't impact simple operators too much like `<` and `>` but it is information you should know for when your building custom validation functions.
+
+With this in mind, we'll first start by giving a minimum and maximum to our value. For simplicity's sake lets go with a minimum of 10 and a maximum of 100.
+
+```javascript
+const validateNumber = (value) => {
+  if (!value) { return undefined; }
+  if (value < 10) {
+    return 'Value should not be less than 10.';
+  }
+  if (value > 100) {
+    return 'Value should not be over 100.';
+  }
+
+  return undefined;
+};
+```
+
+Now for the more complicated validation which is the level of precision we want in our input. For the example let us say that we want 3 decimals worth of precision and we don't want to accept more than that. There are multiple ways of validating this but since our value is a string, we'll use string functions to validate.
+
+We will split the given string on its `.` character, and then check to make sure the 2nd string we get from doing this has a length of 3 or less.
+
+```javascript
+const validateNumber = (value) => {
+  ...
+  const valueSplit = value.split('.');
+  if (valueSplit.length === 2 && valueSplit[1].length > 3) {
+    return 'Value has more than 3 decimal points';
+  }
+
+  return undefined;
+};
+```
+
+This way of validating for precision has a few quirks with it, mainly that if we give it a value of `23.023000` it would fail the validation, because of the extra `0`s at the end despite this number effectively being the same as `23.023`.
+
+For the sake of the example lets try a different way of validating. Let us, instead of using string function to validate, convert the value given to us to a `Number` instead. With it being a number we can use the `Number` class function `toFixed` which allows us to automatically take a value to a given precision.
+
+```javascript
+const validateNumber = (value) => {
+  ...
+  const valueNum = Number(value);
+  if (valueNum.toFixed(3) != valueNum) {
+    return 'Value has more than 3 decimal points';
+  }
+
+  return undefined;
+};
+```
+
+With this we compare the value given us to same value to a given precision. Which means if we are handed `45.4321`, we will compare it to `45.432`, which is not equal and will return the failure message. The quirk with this validation is that if you give it `23.023000` it will succeed because it will be compared to `23.023` and be found equal thus you probably will need to include something to shorten the String value in this case.
```

Comment:
This validation has the opposite quirk. If the input contains extra 0s, like 23.0230000, validation will succeed.

Reply:
Swapped https://github.com/cerner/terra-framework/pull/635/commits/a0241f8008eacb2cb9bba04c762e6018716f0ff5